lib/compat/wordpress-5.9/class-gutenberg-rest-templates-controller.php

```diff
@@ -173,9 +173,9 @@ public function get_item_permissions_check( $request ) {
 	 */
 	public function get_item( $request ) {
 		if ( isset( $request['source'] ) && 'theme' === $request['source'] ) {
-			$template = get_block_file_template( $request['id'], $this->post_type );
+			$template = get_block_file_template( $this->normalize_requested_id( $request['id'] ), $this->post_type );
```

Comment:
Should we use a `sanitize_callback` for id?

Reply:
In the current shape of this PR, yes. But I'm reverting to having `get_template_by_id` with two attempted lookups and it doesn't fit that model unfortunately.